css/style.css

```
@@ -0,0 +1,181 @@
+.bemvindo {
+    height: 500px;
+    width: 1250px;
+}
+
+body {
+    background: #000;
+}
+
+button {
+    background: #FFA500;
+    border: 1px solid #000;
+    color: #000;
+    padding: 5px 20px 5px 20px;
+}
+
+button:hover {
+    background: #000;
+    color: #fff;
+}
+
+.container {
+	display: flex;
+    margin: 0 auto;
+}
+
+#contato {
+    background: #FFA500;
+    padding: 50px;
+    text-align: center;
+}
+
+.fab {
+    color: #000;
+    background: yellow;
+    border-radius: 50%;
+    padding: 10px;
+    margin: 5px;
+}
+
+figcaption {
+    color: #ffa500;
+}
+
+.fundo-slide {
+    width: 100%;
+    height: 400px;
+}
+
+h2 {
+    color: #e6197f;
+    text-align: center;
+}
+
+h3 {
+    color: #FFA500;
+    text-align: center;
+}
+
+h4 {
+    color: #000;
+}
+
+h5 {
+	color: #ffa500;
+}
+
+.img-portifolio {
+	width: 250px;
+}
+
+.img-servicos {
+    border: 2px solid;
+    border-radius: 50%;
+    color: #fff;
+    width: 150px;
+}
+
+.img-slide {
+    position: absolute; 
+	top: 70px; 
+    left: 850px;
+	width: 280px;
+    height: 250px;
+}
+
+.item-portifolio {
+	margin: 20px;
+	background: #000;
+    text-align: center;    
+}
+
+.item-servicos {
+	margin: 50px;
+	background: #000;
+	text-align: center;
+}
+
+.menu ul {
+    width: 100%;
+    height: auto;
+    padding: 10px;
+    margin: 0;
+    background-color: #000;
+    list-style: none;
+  }
+  
+.menu ul li {
+    display: inline-block; /* os itens do menu lado a lado */
+  }
+  
+.menu ul li a {
+    color: #fff;
+    font-size: 14px;
+    text-decoration: none;
+    position: relative; /* O position relative posiciona o elemento em relação a si mesmo. Ou seja, o ponto zero será o canto superior esquerdo, e ele começará a contar a partir dali.*/
+    top: -30px;
+    margin: 10px;
+    padding: 10px;
+  }
+  
+.menu ul li a:hover {
+    border: 1px yellow solid;
+  }
+  
+.menu-logo {
+    width: auto;
+    height: 50px;
+    margin-left: 50px;
+    margin-right: 350px;
+  }
+  
+p {
+    color: #fff;
+    text-align: center;
+}
+
+#portifolio {
+    background: #FFA500;
+    height: 500px;
+    padding: 50px;
+}
+
+.progress {
+    height: 30px;
+    width: 500px;
+    background: #fff;
+    padding: 1px;
+    margin: 20px;
+  }
+  
+.progress-bar {
+    color: #fff;
+    height: 30px;
+    text-align: left;
+    width: 300px;
+    background: #a24168;
+}
+
+.slide {
```

```suggestion
.slide {
    width: 100%;
    display: flex;
    background-image: url(../images/Slider.png);
    height: 400px;
    background-size: 100% auto;
    align-items: center;
    justify-content: center;
}
```

Você pode alinhar os elementos verticalmente usando o próprio `flexbox`. 
Na sugestão acima eu removi o `position relative` e removi também o `fundo-slide` do seu html, adicionando-o como um `background-image`. Mas o que faz o elemento ser alinhado verticalmente é o uso das propriedades `align-itens` e `justify-content`.